doc/rst/users/build.rst

```diff
@@ -7,19 +7,30 @@ Dependencies
 ------------
 
 SCR has several dependencies. A C compiler, MPI, CMake, and pdsh are
-required dependencies. The others are optional, and when they are
+required, as are several component libraries from the ECP VeloC project. The others are optional, and when they are
 not available some features of SCR may not be available.
 
+Required:
+
 * CMake, Version 2.8+
 * Compiler (C, C++, and Fortran)
 * MPI
 * pdsh (https://github.com/grondo/pdsh)
-* DTCMP (optional, used for user-defined directory structure feature)(https://github.com/llnl/dtcmp)
-* LWGRP (optional, used for user-defined directory structure feature) (https://github.com/LLNL/lwgrp)
-* libyogrt (optional, used for determining length of time left in the current allocation) (https://github.com/llnl/libyogrt)
-* MySQL (optional, used for logging SCR activities)
+* AXL (https://github.com/ECP-VeloC/AXL)
+* KVTree (https://github.com/ECP-VeloC/KVTree)
+* er (https://github.com/ECP-VeloC/er)
+* redset (https://github.com/ECP-VeloC/redset)
+* rankstr (https://github.com/ECP-VeloC/rankstr)
+* shuffile (https://github.com/ECP-VeloC/shuffile)
+* spath (https://github.com/ECP-VeloC/spath)
+
+Optional:
+
+* DTCMP (for user-defined directory structure feature)(https://github.com/llnl/dtcmp)
+* LWGRP (for user-defined directory structure feature) (https://github.com/LLNL/lwgrp)
```

Comment:
DTCMP and LWGRP are both now required.  Let's also drop the "for user-defined ..." text.

Reply:
Dropped